### PR TITLE
home page

### PR DESCRIPTION
**Pull request by ndahlquist-cc**
displays game images in descending order, so newest added games are
first seen.  Once we have more games we can increase the number of
images

### CODE REVIEW

File: SeaSharpe_CVGS/SeaSharpe_CVGS/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -1,31 +1,30 @@
...
-            enables a clean separation of concerns and gives you full control over markup
-            for enjoyable, agile development.
-        </p>
-        <p><a class="btn btn-default" href="http://go.microsoft.com/fwlink/?LinkId=301865">Learn more &raquo;</a></p>
-    </div>
-    <div class="col-md-4">
-        <h2>Get more libraries</h2>
-        <p>NuGet is a free Visual Studio extension that makes it easy to add, remove, and update libraries and tools in Visual Studio projects.</p>
-        <p><a class="btn btn-default" href="http://go.microsoft.com/fwlink/?LinkId=301866">Learn more &raquo;</a></p>
+@for (int i = 0; i < Model.Count(); i+=4)
```
Comment (BlackthornYugen) [inline SeaSharpe_CVGS/SeaSharpe_CVGS/Views/Home/Index.cshtml:13]: Does this mean that the controller must sent only multiples of four?

**Comment by ndahlquist-cc** [comment]
yes

**Comment by ndahlquist-cc** [comment]
I could rewrite it so it could send any number, but I think that's time better spent elsewhere

**Comment by BlackthornYugen** [comment]
Was just thinking that it might be good to have a comment in the controller or something but it's not really important. +1

**Pull request merged**